sdk/typespec/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "typespec"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+description = "Project root for all TypeSpec-related crates"
+homepage = "https://aka.ms/typespec"
```

Comment:
```suggestion
homepage = "https://typespec.io"
```

Maybe that aka.ms link should be updated but it links to the telemetry app.

Reply:
Good idea!